app/models/digital_object.rb

```diff
@@ -51,6 +50,8 @@ class DigitalObject < ApplicationRecord
     @mint_doi = false
   end
 
+  # Title
+  metadata_attribute :title, Hyacinth::DigitalObject::TypeDef::Title.new.validation(proc { |value| value&.dig('value').blank? || value.dig('value', 'sort_portion')&.strip.present? })
```

Comment:
I just realized that the downside of doing validations via typedef (rather than regular model validation) is that you can't provide a helpful message about what went wrong.

Reply:
This is true. We could probably refactor to be better about it, but out-of-scope here.

Reply:
👍